### DISCUSSION TITLE
LVM group with LUKS encryption

### DISCUSSION THREAD

**Discussion by kiaraho** [Security-Onion-Solutions/securityonion#2041] [Unsupported Versions] [answered]
When setting up SO2 via the `securityonion-2.3.10.iso` image file, is there an a way to configure the LVM group with LUKS encryption? This is an option during the Ubuntu 18 install (see screenshot below), so I wondering if there was a similar option during the CentOS 7 install.

![image](https://user-images.githubusercontent.com/5502596/99922485-d31adb00-2cfe-11eb-9bbe-35a20dbb7280.png)

**Comment by weslambert** [comment, marked as answer]
You may be able to do something like the following:

https://www.certdepot.net/rhel7-configure-luks-encrypted-partitions-logical-volumes/

 We don't support it OOB from the Security Onion ISO at the moment

**Reply by kiaraho** [reply]
Thanks @weslambert. Any plans to include it from the Security Onion ISO in the future? The steps suggested above would wipe the SO2 installation volume.
